src/styles/transformers/rtl.js

```diff
@@ -0,0 +1,90 @@
+import assign from 'lodash.assign';
+import warning from 'warning';
+
+const reTranslate = /((^|\s)translate(3d|X)?\()(\-?[\d]+)/;
+const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(deg|rad|grad))?/;
+
+/**
+ * This function ensures that `style` supports both ltr and rtl directions by
+ * checking `styleConstants` in `muiTheme` and replacing attribute keys if
+ * necessary.
+ */
+export default (muiTheme) => (style) => {
+  if (process.env.NODE_ENV !== 'production') {
+    warning(!style.didFlip, `You're calling ensureDirection() on the same style
+      object twice.`);
+
+    style = assign({
+      didFlip: 'true',
+    }, style);
+  }
+
+  // Left to right is the default. No need to flip anything.
+  if (!muiTheme || !muiTheme.isRtl) return style;
```

Comment:
@oliviertassinari Let's not limit ourselves for now. We might want to add other things, or if this is gonna be documented, others might like to have access to the theme in their transformers. the impacts of having it is likely none. but excluding it completely might make it hard to put back (or maybe even impossible). besides! we might even be able to use this to pass down userAgent down the context and use it in the prefixer. Just saying :grin: